packages/aragon-cli/src/lib/dao/new.js

```diff
@@ -0,0 +1,38 @@
+const { getRecommendedGasLimit } = require('../../util')
+
+/**
+ * Create a new DAO
+ *
+ * @param {Object} parmas Parameters
+ * @param {Object} repo Template repository
+ * @param {Object} web3 web3
+ * @param {Object} templateInstance Template instance
+ * @param {string} newInstanceMethod New instance method name
+ * @param {string[]} newInstanceArgs New instance arguments
+ * @param {string} gasPrice Gas price
+ */
+module.exports = async function({
+  repo,
+  web3,
+  templateInstance,
+  newInstanceMethod,
+  newInstanceArgs,
+  deployEvent,
+  gasPrice,
+}) {
+  const template =
```

Comment:
I think this part is a bit awkward. Is been more and more clear that we need a clear distinction of how we handle the aragonPM information. Probably something to look a bit more in detail on current iteration after we merge this PR.

Reply:
I agree 💯
The `templateInstance` was originally used here https://github.com/aragon/aragon-cli/blob/develop/packages/aragon-cli/src/commands/run.js#L371 but we'll probably be able to simplify things once we start refactoring the `run` command.